packages/blitz/test/bin.test.ts

```diff
@@ -0,0 +1,15 @@
+import spawn from "cross-spawn"
+
+describe("Binary blitz", () => {
+  it("should return exit code 0 if command is successfull", () => {
+    const result = spawn.sync("blitz", ["--version"])
```

Comment:
I think this also needs to change to `./bin/blitz`. I think just spawning `blitz` might run the globally installed CLI on your machine.

```suggestion
    const result = spawn.sync("./bin/blitz", ["--version"], {stdio: "inherit"})
```